source/catalog.html

```diff
@@ -0,0 +1,259 @@
+          </dl>
+          <a class="additional-product-div__order-button order-button" href="#" role="button">Заказать</a>
+        </div>
+      </li>
+      <li class="additional-product-list__item">
+        <div class="additional-product-div">
+          <h3 class="additional-product-div__product-name"><a class="additional-product-div__link" href="#">Молоко</a>
+          </h3>
+          <dl class="additional-product-div__product-description product-description">
+            <dt class="product-description__product-value">1 литр</dt>
+            <dd class="product-description__product-quantity">100 Р.</dd>
+          </dl>
+          <a class="additional-product-div__order-button order-button" href="#" role="button">Заказать</a>
+        </div>
+      </li>
+      <li class="additional-product-list__item">
+        <div class="additional-product-div">
+          <h3 class="additional-product-div__product-name"><a class="additional-product-div__link" href="#">Витамины</a>
+          </h3>
+          <dl class="additional-product-div__product-description product-description">
+            <dt class="product-description__product-value">1 упаковка (30 г)</dt>
+            <dd class="product-description__product-quantity">300 Р.</dd>
+          </dl>
+          <a class="additional-product-div__order-button order-button" href="#" role="button">Заказать</a>
+        </div>
+      </li>
+    </ul>
+    <div class="sale-block">
+      <p class="sale-block__text">Закажите все и получите чехол для кота в подарок!</p>
+    </div>
+  </section>
+  <section class="company-info">
+    <h2 class="visually-hidden">Карта месторасположения офиса "Кэт Энерджи" и адрес для
+      сотрудничества</h2>
+    <div class="company-info__div">
+      <p class="company-info__lead-text">Приглашаем к сотрудничеству дилеров!</p>
+      <p class="company-info__text">ул. Большая Конюшенная, д. 19/8 Санкт-Петербург</p>
+    </div>
+    <img class="company-info__map map" src="#" alt="Карта по адресу: ул. Большая Конюшенная, д. 19/8 Санкт-Петербург.">
+  </section>
+</main>
+<footer class="page-footer">
+  <a class="logo" href="#"><img class="logo__image logo__image--footer" src="#" alt="Текстовый логотип Кэт Энерджи."></a>
+  <div class="page-footer__div">
+    <ul class="page-footer__social-list social-list">
+      <li class="social-list__item"><a class="social-list__link social-list__link--vk" href="#">Вконтакте</a></li>
+      <li class="social-list__item"><a class="social-list__link social-list__link--instagram" href="#">Инстаграм</a>
+      </li>
+      <li class="social-list__item"><a class="social-list__link social-list__link--facebook" href="#">Фейсбук</a></li>
+    </ul>
+  </div>
+  <div class="page-footer__promo-logo-div">
+    <a class="page-footer__promo-link promo-link" href="https://htmlacademy.ru/intensive/adaptive">HTML Academy</a>
+    <a class="page-footer__promo-link promo-link" href="https://htmlacademy.ru/intensive/adaptive"><img
```

Comment:
```suggestion
    <a class="page-footer__promo-img-link" href="https://htmlacademy.ru/intensive/adaptive"><img
```
чтобы различать эти ссылки как элементы футера